api/src/services/import-services/csv-import-strategy.interface.ts

```diff
@@ -0,0 +1,83 @@
+import { WorkSheet } from 'xlsx';
+import { z } from 'zod';
+import { IXLSXCSVValidator } from '../../utils/xlsx-utils/worksheet-utils';
+
+/**
+ * Type wrapper for unknown CSV rows/records
+ *
+ */
+export type Row = Record<string, any>;
+
+/**
+ * Implementation for CSV Import Services.
+ *
+ * @description All CSV import services should implement this interface to be used with `CSVImportStrategy`
+ */
+export interface CSVImportService<ValidatedRow = Record<string, unknown>, InsertReturn = unknown> {
+  /**
+   * Standard column validator - used to validate the column headers and types.
+   *
+   * @see '../../utils/xlsx-utils/worksheet-utils.ts'
+   * @see '../../utils/xlsx-utils/column-cell-utils.ts'
+   */
+  columnValidator: IXLSXCSVValidator;
+
+  /**
+   * Validate the pre-parsed rows - return either custom Validation or Zod SafeParse.
+   *
+   * @param {Row[]} rows - Raw unparsed CSV rows
+   * @param {WorkSheet} [worksheet] - Xlsx worksheet - useful for calculating non-standard columns
+   * @returns {*} Validation
```

Comment:
nit: return looks like it needs updating

Reply:
fixed